### PR TITLE
Convert wg-k8s-infra to sig-k8s-infra

### PR DESCRIPTION
**Pull request by spiffxp**
Related:
- Part of: https://github.com/kubernetes/community/issues/6036

After nearly 3 years of operation as a WG, the WG K8s Infra organizers would like to propose that WG K8s Infra become a SIG.

What would change:
- convert the existing Organizers to Chairs and Tech Leads
  - Chairs: Arnaud Meukam (@ameukam), Davanum Srinivas (@dims)
  - Tech Leads: Aaron Crickenberger (@spiffxp), Tim Hockin (@thockin)
- transfer ownership of kubernetes/k8s.io from SIG Contributor Experience to SIG K8s Infra as its inaugural subproject
- s/wg-k8s-infra/sig-k8s-infra in a bunch of places
- EDIT: our charter, per request from @kubernetes/steering-committee 

What would stay the same:
- our primary focus would remain the successful migration of all Kubernetes project infrastructure from Google (or elsewhere) to the CNCF, such that the project is able to sustainably operate itself
- continued collaboration with SIGs and subprojects that run apps or services in community-owned project infrastructure
- continued consultation with Steering Committee on policy decisions related to how we spend the community's infrastructure budget

Calls for lazy consensus:
- Proposal: [kubernetes-dev@ thread](https://groups.google.com/g/kubernetes-dev/c/FelvVkgbQrw) (also sent to stakeholder sig mailing lists), lazy consensus deadline: 2021-08-10 @ 12:00 PST (19:00 UTC)
- Charter rewrite first draft: [kubernetes-dev@ thread](https://groups.google.com/g/kubernetes-dev/c/FelvVkgbQrw/m/or_N80VcBgAJ) (also sent to stakeholder sig mailing lists), feedback deadline: 2021-09-10 @ 12:00 PT (19:00 UTC)
- Charter updated, last call: [kubernetes-dev@ thread](https://groups.google.com/g/kubernetes-dev/c/FelvVkgbQrw/m/eTajQz6lCgAJ) (also sent to stakeholder sig mailing lists), lazy consensus deadline: 2021-09-20 @ 12:00 PT (19:00 UTC)

Call for steering approval:
- Proposal: [steering@ thread](https://groups.google.com/a/kubernetes.io/g/steering/c/Y-q0vwKv3bY/m/AyxYChuuAAAJ)

### CODE REVIEW

**Comment by spiffxp** [comment]
/uncc @chris-short @nikhita 

/sig architecture
/cc @derekwaynecarr @dims @johnbelamaric 
sig-architecture chairs

/sig contributor-experience
/cc @alisondy @mrbobbytables 
sig-contributor-experience chairs

/sig release
/cc @justaugustus @saschagrunert 
sig-release chairs

/sig testing
/cc @BenTheElder @stevekuznetsov 
sig-testing chairs

/committee steering
/assign @nikhita 
steering liason

**Comment by thockin** [comment]
FTR: I am +1

**Comment by jberkus** [comment]
FWIW, I never understood why it was a WG in the first place.

**Comment by cpanato** [comment]
+1

**Comment by mrbobbytables** [comment]
+1 from me with my contribex chair hat on

**Comment by rajibmitra** [comment]
+1

**Comment by chris-short** [comment]
+1

**Review by BenTheElder — APPROVED** [review]

**Comment by spiffxp** [comment]
In terms of stakeholder SIGs still waiting to hear from SIG Architecture
- [ ] SIG Architecture (waiting on @derekwaynecarr @dims @johnbelamaric)
- [x] SIG Contributor Experience (@mrbobbytables)
- [ ] SIG Release (@cpanato opted not to +1 as tech lead, waiting on @justaugustus @saschagrunert)
- [x] SIG Testing (@BenTheElder)

**Comment by spiffxp** [comment]
/uncc @BenTheElder @stevekuznetsov @mrbobbytables @alisondy 
since SIG Testing and SIG Contribex have +1'ed

/cc @cpanato @hasheddan @jeremyrickard @puerco 
SIG Release Tech Leads

**Comment by johnbelamaric** [comment]
+1

**Comment by justaugustus** [comment]
Thanks for putting this together, @spiffxp!

Discussed this with @kubernetes/sig-release-leads earlier today and we have two requests:

1. https://github.com/kubernetes-sigs/k8s-container-image-promoter remains a subproject of SIG Release (Release Engineering) with inactive, non-SIG Release subproject owners removed
  
   Given subprojects are intended to be under ownership of a single SIG, we'd just be pruning @kubernetes/wg-k8s-infra-leads from OWNERS/GH repo write/admin teams here.

2. Company diversity in @kubernetes/steering-committee liaisons between SIG Release and (soon-to-be) SIG K8s Infra

   Given SIG Release and SIG K8s Infra [will] have a reasonable amount of control over how this project is released and how infrastructure resources are delegated, we (SIG Release) would like to guard against any potential suggestions of impropriety or steering upon individual contributors, strictly based on what companies they work for.
  
   @dims (SC liaison for SIG Release / incoming Chair for WG K8s Infra) and @nikhita (current SC liaison for WG K8s Infra) work for the same company, and at least from an optics standpoint, it would be good to select a new SC liaison for either SIG.

**Comment by spiffxp** [comment]
>https://github.com/kubernetes-sigs/k8s-container-image-promoter remains a subproject of SIG Release (Release Engineering) with inactive, non-SIG Release subproject owners removed

Given that this proposal doesn't involve changing ownership of this subproject, SGTM

>Given SIG Release and SIG K8s Infra [will] have a reasonable amount of control over how this project is released and how infrastructure resources are delegated, we (SIG Release) would like to guard against any potential suggestions of impropriety or steering upon individual contributors, strictly based on what companies they work for.

There are other SIGs that already violate what you are asking for, and company representation has never been a factor in assigning steering liasons.  Steering and this proposed SIG each individually have employer diversity, so I am having a tough time seeing what additional impropriety is enabled by failing to meet this request

I would welcome anyone interested in improving the diversity of this or any SIG, I just think this is an odd way of going about it.  As a reminder, my chief complaint with WG K8s Infra is that not enough non-Googlers are showing up consistently to do the work.

This sounds like an interesting change to request of Steering to apply consistently to all SIGs, but IMO should not block this proposal.  That said, I will defer to @kubernetes/steering-committee's take on this.

**Comment by mrbobbytables** [comment]
I agree with @spiffxp, to be honest - looking at the company diversity spread of the community groups and that of the steering committee, it'd very difficult to guarantee vendor diversity at the liaison level..especially when we're actively TRYING to encourage more people to take up leadership roles, the chance of a conflict rises greatly. It would also pose other questions like:
- If someone was promoted to a leadership position and suddenly there is a conflict, what would happen?
- What would happen is there is no vendor diversity between the SIG and Steering?
- How do you balance the load of the liaisons? If someone from a company that isn't well represented in the SIGs would they be burdened with being the liaison for most of them?

Stepping back and looking at it from a higher level, if we're trying to prevent potential conflict of interest, what about as an opposing vendor putting up roadblocks for that group? or overtly raising issues? 

Steering is intended to be a trusted community body that is looking out for the project as a whole, and if there isn't some level of trust that the members can set aside their companies when interacting with groups there is a bigger problem at play =/

In any case, if there is still concern, I don’t think this should block the conversion of the WG to a SIG, but could be continued in an issue in k/steering as it’d be a BIG change for all our groups.

**Review by Song2714 — COMMENTED** [review]
# #5928

**Comment by dims** [comment]
+1 from me.

(image-promoter is not part of this PR, so we can deal with it elsewhere)

thanks for doing this @spiffxp

**Comment by derekwaynecarr** [comment]
+1 for me.

**Comment by spiffxp** [comment]
/uncc @cpanato @hasheddan @jeremyrickard @johnbelamaric @justaugustus @saschagrunert @puerco
Apologies for the cc shuffling but github only lets me tag in so many people

Lazy consensus deadline has passed.  

Calling for steering approval: [e-mail to steering@](https://groups.google.com/a/kubernetes.io/g/steering/c/Y-q0vwKv3bY/m/AyxYChuuAAAJ)
/cc @cblecker @liggitt @mrbobbytables @nikhita @parispittman 
- [ ] @cblecker 
- [ ] @derekwaynecarr 
- [ ] @dims 
- [ ] @liggitt 
- [ ] @mrbobbytables 
- [ ] @nikhita
- [ ] @parispittman 

The following steering members +1'ed but I'd like to get confirmation it was as steering vs. stakeholder sig lead:
- @derekwaynecarr 
- @dims 
- @mrbobbytables 
- @nikhita

**Comment by dims** [comment]
+1 (binding - steering)

**Comment by justaugustus** [comment]
> In any case, if there is still concern, I don’t think this should block the conversion of the WG to a SIG, but could be continued in an issue in k/steering as it’d be a BIG change for all our groups.

Thanks for the @kubernetes/steering-committee feedback, @mrbobbytables.
To be clear, this was intended to generate some discussion, not block the conversion.

---

> > https://github.com/kubernetes-sigs/k8s-container-image-promoter remains a subproject of SIG Release (Release Engineering) with inactive, non-SIG Release subproject owners removed

From Aaron:
> Given that this proposal doesn't involve changing ownership of this subproject, SGTM

From Dims:
> (image-promoter is not part of this PR, so we can deal with it elsewhere)

@spiffxp @dims -- Allow me to provide some clarification on why we (SIG Release) thought to include this as a request...

On 7/30, Aaron sent a [note to Steering](https://groups.google.com/a/kubernetes.io/g/steering/c/udk1gQn0q0w/m/zf_uac2DBQAJ) requesting the conversion of WG K8s Infra to a SIG.

On a follow-up to @BenTheElder:

> Was this discussed with any of the SIGs mentioned? Or in the WG?

Aaron said:

> I discussed this privately with stakeholder SIG leads and the WG leads.  The intent was to gain some kind of consensus before opening up for broader review.  This e-mail is that call for broader review: the decision has not been made, merely recommended for further consideration.

Upon discussion (and multiple confirmations) with @kubernetes/sig-release-leads, I found that none of the SIG Release leads had been approached (publicly or privately) by @kubernetes/wg-k8s-infra-leads about this proposal.

The initial [proposal](https://docs.google.com/document/d/1cccUzG1qXIt9Zti_ppwuq6Aw9XDbWPxDigkUiKJF6_s/edit?usp=sharing) sent to Steering had multiple options. To (hopefully faithfully) summarize intents:

- Option 1:
  - convert to a SIG
  - transfer k/k8s.io to new SIG
- Option 2:
  - convert to a Committee, responsible for policy definition around infra
  - transfer k/k8s.io to new Committee
  - SIGs are responsible for handling their infra
- Option 3:
  - convert to a SIG
  - recharter stakeholder SIGs, to include:
  - transfer multiple stakeholder SIG subprojects to this new SIG

It's clear just from the title, and the content therein, that this PR does not move forward with option 2 (Committee).

That said, the first step (convert to SIG) of options 1 and 3 is identical.

In addition, the initial communication to Steering, which includes multiple directions, was elided from this PR's description.

From the perspective of someone who has read the multiple options, this is confusing.
Is this PR option 1 plain and simple, or option 1, with a route to convert to option 3?

Option 1 is pretty clear-cut.

Option 3 is the kind of thing that I'd expect _explicit_ approval from stakeholder SIGs on charter restructuring and subproject transfer (which is why we asked about the image promoter repo).

The idea that we'd be signed up for charter rewrites or subproject transfers without our direct input was worrisome.

From the [creation docs](https://github.com/kubernetes/community/blob/master/sig-wg-lifecycle.md#Creation), WG K8s Infra is following the process to the letter:

- Read [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md)
- Ensure all SIG Chairs, Technical Leads, and other leadership roles are [community members]
- Send an email to the Steering Committee <steering@kubernetes.io> to scope the SIG and get provisional approval
- Look at the checklist below for processes and tips that you will need to do while this is going on. It's best to collect this information upfront so you have a smoother process to launch
- Follow the [SIG charter process](https://github.com/kubernetes/community/tree/master/committee-steering/governance) to propose and obtain approval for a charter
- Announce new SIG on kubernetes-dev@googlegroups.com

SIG Release and @kubernetes/sig-release-leads is **_enthusiastically +1_** to the formation of SIG K8s Infra. We have a set of wonderful set of contributors and leaders in WG K8s Infra doing absolutely mission-critical work for the project.

@kubernetes/steering-committee --

From a wider governance standpoint, we feel we've stumbled into a bit of edge case/process gap.

AFAICT, there's no documented path for conversion of a Working Group to another governance shape. It would be helpful to have future guidance on this, as I'd expect stakeholder SIGs to have input, in some form a bit stronger than simple lazy consensus.

**Comment by spiffxp** [comment]
>Upon discussion (and multiple confirmations) with @kubernetes/sig-release-leads, I found that none of the SIG Release leads had been approached (publicly or privately) by @kubernetes/wg-k8s-infra-leads about this proposal.

I mistook Arnaud for a TL, when he is in fact a Release Manager Associate, and I apologize for the confusion.

>To (hopefully faithfully) summarize intents:

You did a good job of summarizing the options.  My shorter summary of Option 3 would be "initiate a lot of drama" which we decided would be a bad idea.

>The idea that we'd be signed up for charter rewrites or subproject transfers without our direct input was worrisome.

PRs are a great opportunity for direct input!  Note that this PR doesn't rewrite our charter, nor touch anyone else's.

Rest assured we will solicit input from Steering and the SIGs with which we collaborate when we look to rewrite our charter.  From the looks of it there is already a lot that is out of date.  We may find other SIGs will want to consider doing the same.

**Comment by spiffxp** [comment]
>Is this PR option 1 plain and simple, or option 1, with a route to convert to option 3?

To be explicit here: option 1.  But the charter we have is out of date with the state of reality (conversion to a SIG or not) and needs to be rewritten at some point.  It is certainly not my personal intent to boil the ocean and recharter the world when we get to that point.

**Comment by justaugustus** [comment]
Thanks for the apology and clarifications, @spiffxp! 💕

**Comment by cblecker** [comment]
Thanks for putting this together, @spiffxp!

I've added this to the agenda for the next @kubernetes/steering-committee meeting. If we have questions or concerns following that meeting, we will let you know.

/assign
/hold

**Comment by mrbobbytables** [comment]
+1 from me with steering hat on

**Comment by dims** [comment]
thanks for the additional context @justaugustus please do file an issue in kubernetes/steering for the governance (wg->sig) change so we can follow up appropriately.

**Comment by nikhita** [comment]
Based on the discussion, it looks like there are three follow-up/discussion items:

- [ ] Governance concerns w.r.t. steering liaisons - @justaugustus if this is still a concern, can you please file an issue in k/steering for this?
- [ ] Updates to the SIG K8s Infra charter (https://github.com/kubernetes/community/pull/5928#issuecomment-896391670) - @spiffxp can you please create an issue to track this? The issue can be a good avenue to track discussions about splitting ownership of subprojects among SIG K8s Infra and the stakeholder SIGs.
- [x] Need for docs for converting WG to a SIG - I have created https://github.com/kubernetes/steering/issues/215 for this.
- [x] Clarification of steering approval criteria (based on [mailing list discussion](https://groups.google.com/a/kubernetes.io/g/steering/c/Y-q0vwKv3bY)) - I have created https://github.com/kubernetes/steering/issues/214 for this.

**Comment by spiffxp** [comment]
rebased to remove merge conflict

**Comment by cblecker** [comment]
Hey @spiffxp,
As I'm sure you know, with k8s-infra being a working group, by design there's allowed to be some degree with ambiguity with ownership of certain items. It's a temporary effort and things are in flux. With SIGs, the bar is a bit higher as far as the charters are concerned to ensure clear ownership of certain items.. both so that everything has a home, but also so that there isn't questions around which SIG owns a specific policy or piece of code.

In our steering review, the major outstanding point was that some parts of ownership are unclear here and that should be resolved, even if those charters are later iterated on.

Switching over to my ContribEx TL hat, with the proposed relocation of k8s.io/k8s.io over to sig-k8s-infra, there's a number of items that I'm unclear of ownership of. For example:
- The slack-infra tooling configuration includes both the deployment manifests for the tool, as well as the configurations/messaging for things like the welcomer or moderator words. These to me should remain with ContribEx.
- The mailing list configurations in the groups directory should also likely remain with ContribEx, although I'm open to the code responsible for actually reconciling those configurations being owned by sig-k8s-infra
- There are items like the vanity URL redirector config which could go either way as far as ownership
- The apps/prow/ configuration .. is that still with sig-testing, or is that transferring over to k8s-infra?

I think if we can resolve these kinds of grey areas around ownership, there aren't any objections from a steering level to moving this forward.

Thanks!

**Comment by spiffxp** [comment]
I am happy to resolve these things by calling out sub projects within the k8s.io repo more formally. Are you asking for a first pass at this as a blocker to this transition?

**Comment by cblecker** [comment]
Exactly! Some charter language around this would be helpful too.

For example, if the intent is something like "other sigs (ContribEx/Testing/Release/etc) may own the specific configurations for the applications they own, k8s-infra owns the running of those applications" or something along those lines, then adding those applicable items to the in scope vs out of scope sections would clear up some of this ambiguity.

**Comment by spiffxp** [comment]
Ok so to be clear you're asking for a recharter and the formation of new sub projects before you will allow this to merge?

**Comment by cblecker** [comment]
Clarification in the charter as to what is owned by k8s-infra, and what parts remain with other SIGs like ContribEx, Testing, and Release, is a minimum. If the subproject exercise in the k8s.io repo helps with that, then great but not explicitly required as that can be done after the fact to align the subprojects to the intended charter goals.

File: sig-k8s-infra/charter.md
Comment (tpepper) [inline sig-k8s-infra/charter.md:162]: This line reads strangely.

File: sig-k8s-infra/charter.md
Comment (tpepper) [inline sig-k8s-infra/charter.md:87]: nit: "Kubernetes"

File: sig-k8s-infra/charter.md
Comment (cblecker) [inline sig-k8s-infra/charter.md:24]: "project infrastructure" here is quite broad. Is GitHub included in this? Zoom? Slack? Opsgenie?

Comment (spiffxp) [reply sig-k8s-infra/charter.md:24]: I would say: yes.  But this is more specifically handled by existing SIG Contributor Experience subproject(s), so we delegate

Comment (spiffxp) [reply sig-k8s-infra/charter.md:24]: An alternative I think we could draw today would be to say: anything that's IaaS is managed by k8s-infra.  Anything else is managed by a more specific SIG, e.g.
- github, slack, discuss: contribex
- netlify: docs
- artifacthub: is a gap I think? release?
- snyk: security
- travis/circleci/codecov/whatever: is up to each subproject owner?

I like this less since it feels more complex and arbitrary, but I am having difficulty thinking of an exception to this as a guiding principle

**Comment by spiffxp** [comment]
We have made our first attempt at rewriting the charter as part of this PR.  We're asking for input, feedback, and help to avoid perfect becoming the enemy of good here.  Please provide feedback by Friday, Sep 3rd at noon PT.

File: sig-k8s-infra/charter.md
Comment (spiffxp) [reply sig-k8s-infra/charter.md:162]: What I'm trying to get at is
```suggestion
We are not responsible for code that runs _on_ project infrastructure, with the exception of:
- subprojects of this SIG (as listed in the-more-continuously-kept-up-to-date sigs.yaml)
- code we share responsibility for (as listed in the [Cross-cutting and Externally Facing Processes] section)
```

File: sig-k8s-infra/charter.md
Comment (ameukam) [inline sig-k8s-infra/charter.md:104]:
```suggestion
- We are responsible for ensuring community-owned GKE clusters are available
```
We may need more than one cluster.

File: sig-k8s-infra/charter.md
Comment (jeremyrickard) [reply sig-k8s-infra/charter.md:24]: > anything that's IaaS is managed by k8s-infra

I was wondering about this when I read through this originally, especially w/r/t the Areas of responsibility called out below.  I think the bullet points you called out probably makes sense, although `artifacthub`, whatever that ends up being ,  probably should  be  _owned_ by infra, like GCR (or GAR in the future?) is, and then Release would own the artifact promotion, like you called out below? 

If sig-k8s-infra will be the owner of, but will delegate responsibility for, things like Github/Zoom/etc, it would probably be good to explicitly call these out so it is unambiguous?

File: sig-k8s-infra/charter.md
Comment (jeremyrickard) [reply sig-k8s-infra/charter.md:162]: The suggested change is clearer to me.

**Comment by parispittman** [comment]
if my +1 is a percentage it's 55% and the reason why I'm a net positive is because the work needs a permanent home. 
However, from an organizational design perspective at this scale, I think without more work smoothing out sig-testing, sig-contribex, and sig-infra, establishing a SIG may not net you the benefits you are desiring, especially with attracting contributors. This seems like a good short term strategy but not long term. 

I think the three groups are so similar that they deserve to be together for a smoother onboarding process and less administrative overhead. The stuff that doesn't fit might need to be re-thought or restructured anyway to serve a better purpose. We are separating the people from the process and code for a lot of this, especially the contribex bits ie moderation of slack, GitHub, zoom, which opens up to problems that come with silos.

File: sig-k8s-infra/charter.md
Comment (spiffxp) [reply sig-k8s-infra/charter.md:24]: artifacthub = https://artifacthub.io/, ref: https://github.com/kubernetes/steering/issues/188

Comment (dims) [reply sig-k8s-infra/charter.md:24]: @spiffxp haven't we been adding CNCF folks like ihor as owners in different places?

Comment (spiffxp) [reply sig-k8s-infra/charter.md:24]: Good point. Are you suggesting I add collaboration with CNCF in the cross cutting section? Something like ensuring CNCF has sufficient credentials to access all the things as a final fallback?

Comment (spiffxp) [reply sig-k8s-infra/charter.md:24]: Re artifact hub I'm suggesting some part of this project should own the kubernetes org or equivalent there. Not artifact hub itself since it's for all CNCF projects

File: sig-k8s-infra/charter.md
Comment (liggitt) [inline sig-k8s-infra/charter.md:133]: not sure how to parse `Group names that are used to community upon behalf of the project`, rephrase?

**Review by liggitt — COMMENTED** [review]
I thought the enumeration and examples were coherent and helpful in avoiding confusion about where responsibility for a particular thing would fall, thanks. I'm +1 on the shape of this.

File: sig-k8s-infra/charter.md
Comment (cblecker) [reply sig-k8s-infra/charter.md:24]: > anything that's IaaS is managed by k8s-infra

That's much more clear and I would be supportive of this. So our GCP/AWS/other cloud provider accounts would roll up into this sig for now.

File: sig-k8s-infra/charter.md
Comment (cblecker) [inline sig-k8s-infra/charter.md:50]: This one seems like a shared one with Steering/ContribEx

**Comment by hh** [comment]
I'm +1 and look forward to the long running SIG and feel we've done a great job as a pioneering WG.

File: sig-k8s-infra/charter.md
Comment (spiffxp) [reply sig-k8s-infra/charter.md:133]: s/community/communicate, e.g. `contributors@kubernetes.io`

File: sig-k8s-infra/charter.md
Comment (spiffxp) [reply sig-k8s-infra/charter.md:50]: I thought we covered this under the `Public Names` section?  I have no intention of kicking anyone out of OWNERS files so as to ensure policy is enforced.  Ownership of the code I'm gonna push back and suggest that it makes more sense for this SIG to own it since the rest of our infrastructure breaks if this code breaks.

File: sig-k8s-infra/charter.md
Comment (spiffxp) [reply sig-k8s-infra/charter.md]: Made something close to the suggested change: https://github.com/kubernetes/community/pull/5928/files#diff-b55cad1162db37049c36ff01b2892c020ecd9b8ce4a89284850d9bc2b6a6a25aR176-R182

File: sig-k8s-infra/charter.md
Comment (spiffxp) [reply sig-k8s-infra/charter.md]: Found this in a few other places, replaced

File: sig-k8s-infra/charter.md
Comment (spiffxp) [reply sig-k8s-infra/charter.md]: Language that refers to IaaS added here: https://github.com/kubernetes/community/pull/5928/files#diff-b55cad1162db37049c36ff01b2892c020ecd9b8ce4a89284850d9bc2b6a6a25aR28-R34

Language that pushes non-IaaS resources out of scope added here: https://github.com/kubernetes/community/pull/5928/files#diff-b55cad1162db37049c36ff01b2892c020ecd9b8ce4a89284850d9bc2b6a6a25aR184-R187

File: sig-k8s-infra/charter.md
Comment (spiffxp) [reply sig-k8s-infra/charter.md]: Which is basically what I added here: https://github.com/kubernetes/community/pull/5928/files#diff-b55cad1162db37049c36ff01b2892c020ecd9b8ce4a89284850d9bc2b6a6a25aR147-R149

**Comment by spiffxp** [comment]
Updated, PTAL

**Comment by spiffxp** [comment]
@parispittman 
> However, from an organizational design perspective at this scale, I think without more work smoothing out sig-testing, sig-contribex, and sig-infra, establishing a SIG may not net you the benefits you are desiring, especially with attracting contributors. This seems like a good short term strategy but not long term.

Agreed.  But.  Thus far my attempt to change the letters from "WG" to "SIG" has taken over a month and a half.  This does not bode well for the amount of time it would take to have a grand coalescing of all of these SIGs.

Implementing this short term strategy does not preclude a long term strategy.  I would just rather see us get there via incremental progress instead of attempting to boil the ocean.

>I think the three groups are so similar that they deserve to be together for a smoother onboarding process and less administrative overhead. The stuff that doesn't fit might need to be re-thought or restructured anyway to serve a better purpose. 

I'm not sure I follow what you're saying here.  What's the onboarding process we're talking about?  What's the administrative overhead?  What's the stuff that doesn't fit?

> We are separating the people from the process and code for a lot of this, especially the contribex bits ie moderation of slack, GitHub, zoom, which opens up to problems that come with silos.

Everything you've listed is out of scope for this SIG and remains with SIG Contributor Experience, including the code used to manage GitHub and Slack.  I don't follow how this proposal separates people the way you're describing.

**Comment by dims** [comment]
I'm +1 on this. (wearing steering hat)

**Comment by cblecker** [comment]
+1 from me

**Comment by justaugustus** [comment]
Charter updates look great, especially around delegation of responsibilities re: artifacts.
/approve on behalf of @kubernetes/sig-release-leads

**Comment by spiffxp** [comment]
Lazy consensus period has passed

Looking at +1's, /lgtms or /approves since charter was added:
- [x] 4 steering: @dims, @cblecker, @liggitt, @parispittman 
- [x] sig-architecture: @dims
- [x] sig-contributor-experience: @cblecker
- [x] sig-release: @justaugustus 
- [x] sig-testing: @spiffxp

I'm going to rebase to resolve potential merge conflicts, squash commits, and ask for LGTM to merge

**Comment by puerco** [comment]
Congratulations to the new SIG K8s Infra :partying_face: !
/lgtm

**Comment by spiffxp** [comment]
Thanks all for you help in moving this forward!

Now begins a pile of PRs to rename a bunch of things.  Will be tracking in https://github.com/kubernetes/community/issues/6036 if anyone cares to follow along

**Comment by ehashman** [comment]
/honk
:partying_face:

**Pull request merged**
